src/renderer/i18n/index.tsx

```diff
@@ -44,3 +50,32 @@ export async function validateLocale(locale: string): Promise<boolean> {
     locale.startsWith(lang.code)
   );
 }
+
+type TransifexProps<T> = {
+  _str: string;
+  _tags: string;
+} & T;
+
+export function T<Content = {}>({
+  _str,
+  ...props
+}: TransifexProps<Content>): JSX.Element {
+  const node: ReactChild = useT(_str, props);
+  return <>{newlineToBreak(node)}</>;
+}
+
+function newlineToBreak(str: ReactChild): ReactNode {
```

Comment:
IMO it should use *line break* instead of *break*.

```suggestion
function newlineToLineBreak(str: ReactChild): ReactNode {
```

Reply:
IMO the parameter name, `str` should be replaced with `node` or `child`. Its name `str` feels like a string value but the `ReactChild` type is same with `string | number | ReactElement` and it cannot assume the `ReactChild` will be always `string` type.

Reply:
I agree with you. I guess I forgot to rename it while working on it...